Update/yoig_tips_001.txt

```diff
@@ -176,13 +176,13 @@ void main()
 //rその上、ネットでは時々思い出したように新説が立ち上がって話題になり、俺自身もまだ外部に出していない類を山ほど持っている。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "その上、ネットでは時々思い出したように新説が立ち上がって話題になり、俺自身もまだ外部に出していない類を山ほど持っている。",
-			NULL, "Plus, he contributed some topics he found on the net from time to time, and even some material I didn't know myself.", Line_Normal);
+			NULL, "Plus, he contributed some topics he found on the net from time to time - even some material I didn't know myself.", Line_Normal);
 	ClearMessage();
 
 //rゆえに一定期間を置けば『今さら』的な使い古し感もない、便利なネタだった。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "ゆえに一定期間を置けば『今さら』的な使い古し感もない、便利なネタだった。",
-			NULL, "So, there's no point feeling worn out <i>at this point</i> by good news.", Line_Normal);
+			NULL, "So, there's no point feeling worn out by good news <i>now</i>.", Line_Normal);
```

Comment:
The italic "at this point" + being in the middle of the sentence was a little strange.